frontend/packages/topology/src/components/quick-search/QuickSearchModalBody.tsx

```diff
@@ -139,12 +153,17 @@ const QuickSearchModalBody: React.FC<QuickSearchModalBodyProps> = ({
     };
   }, [closeModal, onCancel, onEnter, selectNext, selectPrevious]);
 
+  const getModalHeight = (): number => {
```

Comment:
If you create an items data structure, you can generalize this calculation to be based on the number of item contributors.